lib/sqs.js

```diff
@@ -883,23 +907,25 @@ module.exports = {
 
 /**
  *
- * Type for returned object of `preparedMessages` method
+ * Type for returned object of `preparedMessages` method.
  *
  * @typedef {Object} PreparedMessages
  * @property {Object} Attributes - The attributes for the queue.
  * @property {string} Body - The body of the message.
- * @property {string} MD5OfBody - MD5 of the body of the message.
+ * @property {string|number[]} MD5OfBody - MD5 of the body of the message.
```

Comment:
Where does number[] type comes from?

Reply:
Seems like the returns of md5() are either a string or a number array (byte array). So my Visual Studio Code gave me ts errors with mismatched types. I added number[] to most of the md5 properties initially, but opted to remove them because upon checking the AWS docs, they only mentioned strings. I must've missed this change here, thanks!